eta/core/objects.py

```diff
@@ -42,14 +43,15 @@ class DetectedObject(Serializable, HasBoundingBox):
     '''
 
     def __init__(
-            self, label, bounding_box, confidence=None, index=None, score=None,
-            frame_number=None, index_in_frame=None, attrs=None,
+            self, label, bounding_box, mask=None, confidence=None, index=None,
+            score=None, frame_number=None, index_in_frame=None, attrs=None,
             eval_type=None):
         '''Creates a DetectedObject instance.
 
         Args:
             label: object label string
             bounding_box: a BoundingBox around the object
+            mask: (optional) a mask for the object within its bounding box
```

Comment:
What type is this?

Reply:
clarified that it can be a numpy array of any kind (typically, 2D boolean)